### PR TITLE
Added Serializer.py

### PR DESCRIPTION
**Pull request by GilliusThunderHead**
Filled out base RestAPI framework

### CODE REVIEW

File: freeAgentApp/views.py [hunk 1]
```diff
@@ -81,6 +86,7 @@ class Login(LoginView):
 
 
 
-class LogOut(LogoutView):
+class LogOut(logout):
```
Comment (lemonpkt) [inline freeAgentApp/views.py]: Shouldn't this be `LogoutView` (a built-in generic view class) and not `logout` (a function)?

File: freeAgentApp/views.py [hunk 2]
```diff
@@ -81,6 +86,7 @@ class Login(LoginView):
...
     # next_page = 'login'
+    def logout (self,request):
     pass
```
Comment (lemonpkt) [inline freeAgentApp/views.py]: I think `pass` would need to be indented here

File: freeAgentApp/serializer.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializer
+from .models import Project,Review
```
Comment (lemonpkt) [inline freeAgentApp/serializer.py]: I think this needs an `import UserProfile` so that `class UserProfileSerializer` will work properly

**Comment by GilliusThunderHead** [comment]
Your are correct. I fixed these this morning. Not sure about the LogoutView though

Sent from my iPhone

On 7 Oct 2017, at 11:50 am, Alex Mueller <notifications@github.com<mailto:notifications@github.com>> wrote:


@lemonpkt commented on this pull request.

Good work, but I'd just like clarification on a couple of things which I've left comments about.

________________________________

In freeAgentApp/views.py<https://github.com/lemonpkt/IPProject/pull/39#discussion_r143317544>:

> @@ -81,6 +86,7 @@ class Login(LoginView):



-class LogOut(LogoutView):
+class LogOut(logout):


Shouldn't this be LogoutView (a built-in generic view class) and not logout (a function)?

________________________________

In freeAgentApp/views.py<https://github.com/lemonpkt/IPProject/pull/39#discussion_r143317569>:

>      # next_page = 'login'
+    def logout (self,request):
     pass


I think pass would need to be indented here

________________________________

In freeAgentApp/serializer.py<https://github.com/lemonpkt/IPProject/pull/39#discussion_r143317598>:

> @@ -0,0 +1,20 @@
+from rest_framework import serializer
+from .models import Project,Review


I think this needs an import UserProfile so that class UserProfileSerializer will work properly

**Comment by lemonpkt** [comment]
@GilliusThunderHead Cool, you'll need to push the updated commit so we can merge it in. With the logout view I mean that I think this:

```python
class LogOut(logout):
```

should be this instead:

```python
class LogOut(LogoutView):
```
If you can quickly test to make sure logging out still works and let me know that'd be great.

**Comment by GilliusThunderHead** [comment]
Sorry, misunderstood, it is LogoutView on the branch I'm working on so Gary may have changed it. I think we may have to look at UserProfile as the auth_user has been overridden by it maybe? Having trouble calling the current user in IndexView.

Sent from my iPhone

On 7 Oct 2017, at 12:26 pm, Alex Mueller <notifications@github.com<mailto:notifications@github.com>> wrote:


@GilliusThunderHead<https://github.com/gilliusthunderhead> Cool, you'll need to push the updated commit so we can merge it in. With the logout view I mean that I think this:

class LogOut(logout):

should be this instead:

class LogOut(LogoutView):

If you can quickly test to make sure logging out still works and let me know that'd be great.

**Pull request merged**